packages/css/src/components/form/form-groups/static/form-group-static.scss

```diff
@@ -19,14 +19,17 @@
       }
     }
 
-    &.dso-edit .dso-field-container {
-      padding-right: units.$u5;
-      position: relative;
+    &.dso-edit,
+    &.has-edit {
```

Comment:
@daverhelst is dit jouw change?

Reply:
Klopt, er zat een discrepantie in de .njk file en de .template.ts van `form-group-static`, maar nu ik er nog eens naar kijken zou de templating in `group-static.njk` gewoon rechtgetrokken moeten worden. Dat heb ik nu gedaan

Reply:
Dit hebben we samen opgelost